step-release-vis/src/app/models/Data.ts

```diff
@@ -7,3 +7,7 @@ import {
 export type Environment = EnvProto.AsObject;
 export type CandidateInfo = CandidateInfoProto.AsObject;
 export type Snapshot = SnapshotProto.AsObject;
+export interface Timestamp {
```

Comment:
can we use this directly from the proto instead?

Reply:
Done